src/NATS.Client.Core/INatsConnection.cs

```diff
@@ -97,6 +115,8 @@ public interface INatsConnection
     ValueTask<NatsMsg<TReply?>?> RequestAsync<TRequest, TReply>(
         string subject,
         TRequest? data,
+        string? queueGroup = default,
```

Comment:
I don't think `queueGroup` would ever get used - since `Request` makes a new inbox and doesn't expose it, the subscribe that it creates is likely the only subscriber that will ever listen, so there is not a need for a work queue

Reply:
Good catch. Removed.